EpsilonLights/Src/Lights.c

```diff
@@ -7,90 +7,89 @@ void updateLightsTask(void const* arg)
     // One time osDelayUntil intialization
     uint32_t prevWakeTime = osKernelSysTick();
     // Store inputs values
-    char headlightsOff;
-    char headlightsLow;
-    char headlightsHigh;
-    char rightSignal;
-    char leftSignal;
-    char hazards;
-    char brakes;
-    char bmsStrobe;
-    uint32_t regenBrakeInt;
-    float regenBrakeFloat;
+
+
+    lightsInfo lightsCharacteristics;
```

Comment:
I would name the variable the same as the struct name. So `LightsInfo lightsInfo`.